$NetBSD: patch-ah,v 1.7 2016/08/21 11:39:31 wiz Exp $

mutt_curses.h refers to event_t from mutt,h, and SHORT_STRING from lib.h,
so include those headers here.

--- mutt_curses.h.orig	2016-08-11 01:47:00.000000000 +0000
+++ mutt_curses.h
@@ -52,6 +52,9 @@
 # include <curses.h>
 #endif
 
+#include "lib.h" /* for SHORT_STRING */
+#include "mutt.h" /* for event_t */
+
 #define MUTT_ENTER_C '\n'
 #define MUTT_ENTER_S "\n"
 
